home.html

```diff
@@ -7,6 +7,66 @@
     <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-QWTKZyjpPEjISv5WaRU9OFeRpok6YctnYmDr5pNlyT2bRjXh0JMhjY6hW+ALEwIH" crossorigin="anonymous">
 </head>
 <body>
+  <header>
+    <!-- Navbar Start -->
+    <nav class="navbar container navbar-expand-lg">
+        <div class="container-fluid px-0">
+            <div class="rectangle-nav"></div>
+            <div class="collapse navbar-collapse" id="navbarSupportedContent">
+                <ul class="navbar-nav ms-auto mb-lg-0">
+                    <li>
+                        <a class="nav-link mx-2" href="#description">Our Services</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link mx-2" href="#why-us">Why Us</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link mx-2" href="#testimonial">Testimonial</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link mx-2" href="#faq">FAQ</a>
+                    </li>
+                    <li class="nav-item">
+                        <button class="btn btn-success mx-2">Register</button>
+                    </li>
+                </ul>
+            </div>
+        </div>
+    </nav>
+    <!-- Navbar End -->
+</header>
+<main>
```

Comment:
Remove the <main> tag or add the closing tag </main> to close the main content section of the page.